cmd/list_bundles_test.go

```diff
@@ -1,15 +1,69 @@
 package cmd
 
 import (
+	"os"
+	"path/filepath"
+	"strings"
+
 	. "github.com/onsi/ginkgo/v2/dsl/core"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
 )
 
 var _ = Describe("List Bundles Cmd", func() {
 	When("Calling opcap list bundles", func() {
+		var tmpDir string
+		var gitUrl string
+		var err error
+
+		// create fake test data before each test case
+		BeforeEach(func() {
+			AppFs := afero.NewMemMapFs()
+			tmpDir, err = createFakeRepo(AppFs)
+			Expect(err).ToNot(HaveOccurred())
+			DeferCleanup(os.RemoveAll, tmpDir)
```

Comment:
This should be unnecessary. Once the MemMapFs goes out of scope, it will be thrown away. With that said, It seems that AppFs would go out of scope here? Does this function as you expect?